psl-cli/src/main/java/org/linqs/psl/cli/Launcher.java

```diff
@@ -442,7 +442,7 @@ private void run() {
 
         if (parsedOptions.hasOption(CommandLineLoader.OPTION_VISUALIZATION)) {
```

Comment:
There are still options which indicate visualization instead of data collection.